src/Renderer/Twig/Extension.php

```diff
@@ -296,15 +302,17 @@ public function createUrl($value = null, $options = null): ?string
                         $url .= '?'.$hash;
                     }
                     $url = $base.'/'.ltrim($url, '/');
-                } else {
+                } else { // others cases
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method createUrl uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/app/Narno/Cecil/pullRequest?prid=4576548)